### ISSUE TITLE
Move SetFogFilter to be part of FOWOptions

### ISSUE THREAD

**Issue by Solaxi** [AsyncTI4/TI4_map_generator_bot#2681]
^

**Comment by Solaxi** [comment]
Or wait no. It's per player setting... Move it to be User command... or something. Should not be under regular /fow.

**Comment by jamesrudolph** [comment]
Does /map have room for the map making commands? (25 subcommands max)

**Comment by Solaxi** [comment]
Yeah /map has room. Also I'd like to remove RemoveAllAdjacencyOverrides and RemoveAllCustomAdjacentTiles and make RemoveAdjacencyOverride/RemoveCustomAdjacentTile support that using primary_tile:ALL.
